examples/benchmark/benchmark.js

```diff
@@ -21,7 +21,9 @@ suite
     uuidv1();
   })
   .add('uuidv1() fill existing array', function () {
-    uuidv1(null, array, 0);
+    try {
+      uuidv1(null, array, 0);
+    } catch (err) {}
```

Comment:
I think in the benchmark it is acceptable to just swallow the error.

However just reading this piece of code without additional knowledge might be really confusing. Please add a comment that explains why you are catching the error:

```js
    // The spec (https://tools.ietf.org/html/rfc4122#section-4.2.1.2) defines that only 10M/s v1
    // UUIDs can be generated on a single node. This library throws an error if we hit that limit
    // (which can happen on modern hardware and modern Node.js versions).
```